src/components/v5/common/ActionSidebar/partials/forms/EditColonyDetailsForm/partials/SocialLinksTable/hooks.tsx

```diff
@@ -53,29 +53,34 @@ export const useSocialLinksTableColumns = (): ColumnDef<
 export const useGetTableMenuProps = (
   { remove },
   setSocialLinkIndex: (index: number) => void,
+  isReadOnly?: boolean,
 ) => {
   return useCallback<
     TableWithMeatballMenuProps<SocialLinksTableModel>['getMenuProps']
   >(
-    ({ index }) => ({
-      cardClassName: 'min-w-[9.625rem] whitespace-nowrap',
-      items: [
-        {
-          key: 'edit',
-          onClick: () => {
-            setSocialLinkIndex(index);
-          },
-          label: formatText({ id: 'table.row.edit.link' }),
-          icon: 'edit-pencil',
-        },
-        {
-          key: 'remove',
-          onClick: () => remove(index),
-          label: formatText({ id: 'button.delete' }),
-          icon: 'trash',
-        },
-      ],
-    }),
-    [setSocialLinkIndex, remove],
+    ({ index }) => {
```

Comment:
I'm not a huge fan of this solution, could we maybe do this a level higher? I know it's a hook currently, but it doesn't need to be one, create a `getTableMenuProps` function which is either called and the data used, or we use `undefined` in `SocialLinksTable`